packages/home-spa/src/js/controller.js

```diff
@@ -1,12 +1,18 @@
 // Services
 import { getData }  from './service';
 import { stub } from '../mocks/stub';
+// importing images
+import '../img/bell.svg';
+import '../img/shield.svg';
+import '../img/talk-bubble.svg';
+import '../img/login.svg';
+import '../img/magnifying-glass.svg';
 
 const microserviceCard = document.getElementById('microservice-cards');
-const spaList = document.getElementById('banner__links');
 const carouselSlide = document.getElementById('carousel-slide');
+const spaList = document.getElementById('banner__links');
 
-microserviceCard.innerHTML = `<div class="loader"></div>`;
+microserviceCard.innerHTML = `<div></div><div class="loader"></div>`;
```

Comment:
Why is there an empty `div` here?

Reply:
To position the loader in the center as its a grid the loader takes up the first gird block

Reply:
You can use the grid-column/grid-row (whichever fits best) css property to place the div in the right spot.

Reply:
Required styles added and extra div removed